test_settings.py

```diff
@@ -33,6 +33,7 @@ def root(*args):
     'django.contrib.messages',
     'django.contrib.sessions',
     'edx_event_bus_kafka',
```

Comment:
Are the changes in this file necessary?

Reply:
Nope, but it helps when you want to run `consume_events` command directly from this repo while testing locally.